src/main/kotlin/no/item/gradle/beans/GeneratedField.kt

```diff
@@ -0,0 +1,10 @@
+package no.item.gradle.beans
```

Comment:
Are `data classes` really `beans`? I would possibly name this package `models`.

  - no.item.xp.plugin
  - no.item.xp.plugin.models
  - no.item.xp.plugin.parsing
  - no.item.xp.plugin.writer

What do you think?

Reply:
No they are not beans. Models are a better name,  will be changed.